documentation/docs/14-configuration.md

```diff
@@ -219,3 +224,6 @@ Whether to remove, append, or ignore trailing slashes when resolving URLs to rou
 ### vite
 
 A [Vite config object](https://vitejs.dev/config), or a function that returns one. Not all configuration options can be set, since SvelteKit depends on certain values being configured internally.
+
+### vitePluginSvelteConfig
+A [vite-plugin-svelte config object](https://github.com/sveltejs/vite-plugin-svelte/blob/main/docs/config.md). These options must be in root scope of config object in svelte.config.js file.
```

Comment:
This also feels a bit out of place. The list of options here implies `config.kit.*` but this section is for the spread expression only. I think we can remove this too with the suggestion above.